libs/sdk-ui-dashboard/src/presentation/presentationComponents/DashboardItems/DashboardItem.tsx

```diff
@@ -20,18 +20,48 @@ const screenClasses: { [S in ScreenSize]: string } = {
     xl: "layout-xl",
 };
 
+const HiddenDescription = ({ children, id }: { children: React.ReactNode; id: string }) => (
+    <span
+        id={id}
+        style={{
```

Comment:
Can you please extract this as a separate object outside of the component in the next PR?